cityofaustin/atd-vz-data
# add micromobility to vzv chart

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/12005

## Testing
**URL to test:** 
[vzv](https://deploy-preview-1261--atd-vzv-staging.netlify.app/)
[vze](https://deploy-preview-1261--atd-vze-staging.netlify.app/)

**Steps to test:**
- Open VZV and confirm that the By Mode chart displays the new 'E-Scooter Rider' category with a new symbol and color from our existing color palette (we went with a phone icon because font awesome unfortunately doesn't have a scooter icon)
- References to this mode in the VZE fatalities and crashes tables should be consistent as well!

---
#### Ship list
